src/server/scripts/EasternKingdoms/BlackrockMountain/BlackrockSpire/boss_the_beast.cpp

```diff
@@ -100,10 +220,69 @@ class boss_the_beast : public CreatureScript
             }
             DoMeleeAttackIfReady();
         }
+
+        void FindNearbyOrcs()
+        {
+            std::vector<Creature*> temp;
+            me->GetCreatureListWithEntryInGrid(temp, NPC_BLACKHAND_ELITE, 50.0f);
+
+            for (Creature* creature : temp)
+                nearbyOrcsGUIDs.push_back(creature->GetGUID());
+        }
+
+    private:
+        bool beastReached;
```

Comment:
private members should have the _ prefix. Nice job btw 😄

Reply:
I think the new script register model needs to be used for all the new scripts at least. If you have the time, you could also convert those you are updating haha

Reply:
aight